script/lazytox.py

```diff
@@ -0,0 +1,204 @@
+#!/usr/bin/env python
+"""
+Lazy 'tox' to quickly check if branch is up to PR standards.
+
+This is NOT a tox replacement, only a quick check during development.
+"""
+import os
+import asyncio
+import sys
+import re
+from collections import namedtuple
+
+RE_ASCII = re.compile(r"\033\[[^m]*m")
+ERROR = namedtuple('ERROR', "file line col msg")
```

Comment:
Python classes are usually UpperCamelCase to avoid confusion with constants.

```python
Error = namedtuple('Error', "file line col msg")
```

Reply:
Will change and add pylint disable directive